yarn.lock

```diff
@@ -534,9 +534,9 @@
     ulid "^2.3.0"
 
 "@aws-amplify/geo@console-preview":
-  version "3.0.1-console-preview.11cf989.0"
-  resolved "https://registry.npmjs.org/@aws-amplify/geo/-/geo-3.0.1-console-preview.11cf989.0.tgz#0a08f01ae65284d45d37e81abbafcbd8b468125f"
-  integrity sha512-O7GAqA+J37Byi7B0tAkhzi7cnC3MvAfX1bCBhlA0Rqt43mQpzHY0lW+APMvogLxWSkTGotTWXA0HbjeCvjOnXg==
+  version "3.0.1-console-preview.b278dcb.0"
```

Comment:
The rest of the v6 deps have the same hash (`console-preview.8d88eef.0+8d88eef`), can you please run `yarn clean && yarn` so they all update to the same version?